api/database/factories/PoolCandidateSearchRequestFactory.php

```diff
@@ -29,8 +30,8 @@ public function definition()
     {
 
         $communityFetched = Community::inRandomOrder()->first();
-
         $community = isset($communityFetched) ? $communityFetched : Community::factory()->create();
+        $user = User::inRandomOrder()->first();
```

Comment:
I'd recommend having some of these seed null for realism.

Reply:
Good call! Fixed in [bb360d9](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11275/commits/bb360d9a772ebe3a4d79357f56c91a947f4a8127)